src/librustc_interface/util.rs

```diff
@@ -183,15 +183,15 @@ pub fn spawn_thread_pool<F: FnOnce() -> R + Send, R: Send>(
 
     let with_pool = move |pool: &ThreadPool| pool.install(move || f());
 
-    syntax::with_globals(edition, || {
-        syntax::GLOBALS.with(|syntax_globals| {
+    syntax::attr::with_globals(edition, || {
```

Comment:
This just doesn't look good.
The fact that libsyntax globals have some attribute-related fields is an implementation detail.
Perhaps the globals need to be put into their old place or at least publicly reexported from their old place.

Reply:
My thinking is that this would eventually be moved to rustc_attr, but re-exporting them for now seems fine as it is relatively non-invasive.